Add showPopperArrow prop

somewhatabstract opened the pull request:
Addresses #1029 

This adds a new prop called `showPopperArrow` and corresponding example to the docs.

The property defaults to `true` to ensure backwards compatibility. Setting it to false results in the popper anchor arrow not being displayed.

![image](https://user-images.githubusercontent.com/1266297/64053886-085f6a00-cb4a-11e9-824a-769377e318d6.png)

martijnrusschen commented:
Awesome, can you perhaps add a few tests to make sure the behavior doesn't break in the future?

somewhatabstract commented:
> Awesome, can you perhaps add a few tests to make sure the behavior doesn't break in the future?

Sure. I wasn't quite sure what tests to add. The ideal thing would be a snapshot test, but I don't think y'all support that right now. Do you have a suggestion? In the meantime, I'll poke at something and see where I get.

somewhatabstract commented:
OK, I've added a couple of test cases. Let me know if these are insufficient and what additional tests you'd like to see. 😄

martijnrusschen commented:
Yep, this is sufficient. Can you resolve the merge conflict, and I'll merge it when it's ready.

somewhatabstract commented:
OK @martijnrusschen, I have rebased this PR onto the latest `master` and addressed the merge conflict. I think this is all set. Please let me know if there's anything else I can do to help, with this PR or anything else :)
